change message templates for phonenumber validator

daschJulia opened the pull request:
use readable message template to show the correct violation
of a phone number in our context.

rkcpi commented on src/main/java/net/contargo/types/truck/BulgarianLicensePlateHandler.java:
```diff
@@ -0,0 +1,80 @@
+ *
+ * <ul>
+ *   <li>A-1234-BB</li>
+ *   <li>AAQ-1234-BB</li>
```
This is wrong. The wiki page says it starts either with a *one* or *two* letter province code. Not three as in this example.

vanherpen replied:
Onm typen the second A I accidently also touched the Q. Q should'nt be there.

rkcpi commented on src/main/java/net/contargo/types/truck/BulgarianLicensePlateHandler.java:
```diff
@@ -0,0 +1,80 @@
+        String normalizedValue = normalize(value);
+
+        // A-1234-AA
+        boolean leadingLetter = normalizedValue.matches("[A-Z]\\-[1-9]{4}\\-[A-Z]{2}");
```
Maybe it might be more performant to statically initialize the regex as a `Pattern`.

daschJulia committed:
change message templates for phonenumber validator

use readable message template to show the correct violation
of a phone number in our context.

rjayasinghe reviewed (approved):
great! :)

rjayasinghe commented:
@contargo-development please merge
